src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -278,7 +280,7 @@ def handle_message(self, message: Message) -> None:
                         return
             except QuerySubscription.DoesNotExist:
                 metrics.incr("snuba_query_subscriber.subscription_doesnt_exist")
-                logger.error(
+                logger.warning(
```

Comment:
Just fixed a couple of logging things while I'm in here. This isn't really an error, it's common that someone deletes an alert rule and one more update comes through due to timing issues